packages/edit-site/tsconfig.json

```diff
@@ -0,0 +1,56 @@
+{
+	"$schema": "https://json.schemastore.org/tsconfig.json",
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types"
+	},
```

Comment:
This becomes redundant after ca616014255efbb61f34c10917d52a2d86c1c660 (rebase required):

```suggestion
```